src/aliases/parser/alias_parser.go

```diff
@@ -0,0 +1,15 @@
+package aliasparser
+
+import (
+	"context"
+
+	"github.com/consensys/quorum-key-manager/src/aliases"
+)
+
+//go:generate mockgen -source=alias_parser.go -destination=mock/alias_parser.go -package=mock
+
+// AliasParser parses and replace aliases.
+type AliasParser interface {
```

Comment:
To be honest I liked better your proposal to completely remove this and to add these use cases to the connectors. IMO you just created this component because of the way Nico created the tickets: https://app.zenhub.com/workspaces/tanuki-5ea70772b186e10067f57842/issues/consensys/quorum-key-manager/148 `(Allow to pass alias placeholders)`

That's why you separated it in a different component and called it "placeholder", but fundamentally this just goes into your AliasBackend.